src/core/imap/MCIMAPSession.cpp

```diff
@@ -688,12 +688,12 @@ void IMAPSession::connect(ErrorCode * pError)
         break;
 
         case ConnectionTypeTLS:
+#if defined(_WIN32)  || defined(_WIN64)
```

Comment:
```suggestion
#if _MSC_VER
```
That would be enough. Such check is used in other places of mailcore.